lib/game.rb

```diff
@@ -29,46 +31,48 @@ def set_options
       @human_player.marker = @marker
     end
 
+    def generate_markers
```

Comment:
I pass in all players' markers when determining the game's outcomes. Originally, I did not need to do this when those game state methods existed in `Game` since I had instance variables that I could refer to.